EvotoApi/Areas/Management/ManagementApiAreaRegistration.cs

```diff
@@ -19,6 +19,12 @@ public override void RegisterArea(AreaRegistrationContext context)
                 "mana/{controller}/{action}/{id}",
                 new { action = "Index", id = UrlParameter.Optional }
             );
+
+            context.MapRoute(
```

Comment:
There should either be static routing (using route attributes [RoutePrefix("test")] and [Route("subTest")] )

Which should override any mapped routes. Then route mapping is used when static routes aren't defined or found for the current route. 

If there is an issue, it could be caused by the order of things in WebApiConfig. I tweaked it a bit in the registrar api.
[RegistrarApi.WebApiConfig](https://github.com/evoto-tech/EvotoApi/blob/master/RegistrarApi/App_Start/WebApiConfig.cs)
[ManagementApi.WebApiConfig](https://github.com/evoto-tech/EvotoApi/blob/master/EvotoApi/App_Start/WebApiConfig.cs)

Just leave it for now then, I'll take a look when I refactor the projects